etl/permissions.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import permissions
+
+
+class ETLPermissions(permissions.BasePermission):
```

Comment:
Check the docstring - has remnants of organization. Also, why should we keep has_permission as True and not keep it restricted to superuser as well?